chart/monocular/values.yaml

```diff
@@ -148,5 +148,10 @@ mongodb:
 # This must be set if mongodb.enabled is set to false, following the pattern:
 # `mongodb://${MONGODB_USER}:${MONGODB_ROOT_PASSWORD}@${MONGODB_DNS}:${MONGODB_PORT}/${MONGODB_DATABASE}`
 # ref: https://docs.mongodb.com/manual/reference/connection-string/
+# You may set the connection URL in one of two ways:
+# - mongoUrl: store the connection string directly in Helm values
+# - mongoUrlSecret: name of the secret where the connection string is stored,
+#   where the key is "mongo-url-secret". See NOTES.txt.
 global:
```

Comment:
What do you think about moving this configuration to `externalDatabase`? e.g. https://github.com/helm/charts/blob/master/stable/wordpress/values.yaml#L78. Then in the future we are one step closer to splitting up the options in host, user, password, etc. It would make this a breaking change, but I doubt anyone is using this setting yet so the risk should be very low.